src/components/charts/LineMergeTimeline.js

```diff
@@ -18,7 +18,7 @@ class LineMergeTimeline extends Component {
 
   constructor(props) {
     super(props);
-    const {startTime, endTime} = getStartAndEndTime(
+    const {startTime, endTime} = Array.isArray(this.props.timeData) && getStartAndEndTime(
```

Comment:
isValidation(props) {
  const { timeData, lineData } = this.props
  if (isEmpty(timeData) || isEmpty(lineData) return 'haveData'
  if (!Array.isArray(timeData) || Array.isArray(lineData)) return 'typeOfVariable'
}

constructor(props) {
  super(props);
   if (!isValidation()) return;

componentDidMount = () => {
   if (!isValidation()) return;
   ....
}

render = () => {
   if (!isValidation()) return <div>{errorMessage(isValidation())}</div>
   ....
}